packages/runtime/test/lib/testUtils.ts

```diff
@@ -733,6 +735,36 @@ export async function executeFullRequestAndShareThirdPartyRelationshipAttributeF
     return ownSharedThirdPartyRelationshipAttribute;
 }
 
+export async function executeFullShareThirdPartyRelationshipAttributeFlow(
```

Comment:
this method name is **really** misleading. As far as I understand it you could use it to share and accept every kind of ShareAttributeRequestItem and not only third party related stuff.